x-pack/plugin/sql/sql-client/src/main/java/org/elasticsearch/xpack/sql/client/HttpClient.java

```diff
@@ -137,9 +139,9 @@ private boolean head(String path, long timeoutInMs) throws SQLException {
         return fromXContent(response.v1(), response.v2(), responseParser);
     }
 
-    private static <Request extends ToXContent> byte[] toXContent(Request xContent) {
+    private <Request extends ToXContent> byte[] toXContent(Request xContent) {
```

Comment:
Very minor: Personally I would keep it static and pass the contentType as parameter.

Reply:
I find it cleaner this way. Will keep it as is.